src/pages/guides/using/asset-events/asset-events-available.md

```diff
@@ -0,0 +1,8 @@
+# Available Asset Events
+
+Asset Events are sent for the following [resources](asset-events-glossary.md#resource):
+
+| Resource                                                                                                   | Applicable Assets          | Actions                |
+| ---------------------------------------------------------------------------------------------------------- | -------------------------- | ---------------------- |
+| [Primary](asset-events-glossary.md#primary-resource)                 | File, Directory, Composite | Create, Update, Delete |
+| [Repository Metadata](asset-events-glossary.md#repository-metadata-resource) | File, Directory, Composite | Create, Update, Delete |
```

Comment:
TBH, I am not sure if this page adds any value. It may confuse the users.

Reply:
This does add value when you have to map from the previous payload to the new one. if you consider the two payloads side by side for an action, it helps.

Reply:
I am confused. Don't we already have the `asset-events-migration.md` page for that?

Reply:
oh sorry, i got confused. I agree. we can remove this page. it will confuse.